sympy/physics/vector/point.py

```diff
@@ -93,6 +94,12 @@ def a1pt_theory(self, otherpoint, outframe, interframe):
         >>> O.set_vel(N, 0)
         >>> P.a1pt_theory(O, N, B)
         (-25*q + q'')*B.x + q2''*B.y - 10*q'*B.z
+        >>> a, b = symbols('a b', cls=Point)
+        >>> type(a)
+        <class 'sympy.physics.vector.point.Point'>
+        >>> x = Point('X')
+        >>> type(a) == type(x)
+        True
```

Comment:
Make it a separate example, not mixed in with the first one.

Reply:
Right, I'll close this PR, and open a new one for the Point docstring and `docs/` update. Thanks!

Reply:
You don't need to close this PR, just add commits.